sale_product_set/views/product_set.xml

```diff
@@ -72,8 +72,8 @@
         <field name="view_id" ref="view_product_set_tree"/>
     </record>
 
-    <menuitem id="menu_product_set"
-        parent="sales_team.menu_sales"
+    <menuitem id="menu_product_set_config"
+        parent="sale.menu_sales_config"
```

Comment:
Is it Config or Catalog?  Just an opinion but I'd say it is more catalog. - non blocking

Reply:
sounds more like config but I have no constrain... ;)